cap-ci/pipeline.yaml.tmpl

```diff
@@ -80,6 +80,24 @@ resources:
     endpoint: null
 {{ end }}
 
+{{- if index $config.jobs "pre-upgrade-deploy-kubecf" }}
```

Comment:
If I understand right, these won't work when we're upgrading and the chart repo contains the same version as we're upgrading to. Instead, we should set the version explicitly in the chart resource (from config set in the upgrade pipeline yamls)

Reply:
Actually if we merge https://github.com/SUSE/catapult/pull/274 , we could just skip using the helm-chart concourse resource altogether

Reply:
` these won't work when we're upgrading and the chart repo contains the same version as we're upgrading to` Thats correct. But this work is for only 2 scenarios. 
1. cap upgrade from released helm chart to nightly master
2. cap upgrade from release helm chart to rc

In both cases, chart repo will not be the same version as we are upgrading to and we do not want to use chart repo for installation if its not a "cap release upgrade pipeline" or "cap pre release upgrade pipeline". Any other scenario will be its own custom pipeline definition

Reply:
`we could just skip using the helm-chart concourse resource altogether`: I have made a deliberate decision to use a helm chart concourse resource as it makes things clearer on concourse UI about the version picked up and have a same view as we get in non-upgrade CIs (cap release and cap pre release) and stratos and metrics installation.